examples/JSX-example/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
why we has change of JSX example here, I thought these change should belong to PR #5

Reply:
Fixed